azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/calling/controlbar/more/MoreCallOptionsListViewModel.kt

```diff
@@ -18,11 +27,33 @@ internal class MoreCallOptionsListViewModel(
 
     val displayStateFlow = MutableStateFlow(false)
 
+    val listEntries = mutableListOf<Entries>().apply {
+        add(Entries.SHARE_DIAGNOSTICS)
+        if (showSupportFormOption) {
+            add(Entries.REPORT_ISSUE)
+        }
+    }
+
     fun display() {
         displayStateFlow.value = true
     }
 
     fun close() {
         displayStateFlow.value = false
     }
+
+    fun requestReportIssueScreen() {
+        dispatch(NavigationAction.ShowSupportForm())
+    }
+
+    companion object {
+        enum class Entries(val title: Int, val icon: Int?,) {
+            SHARE_DIAGNOSTICS(
+                R.string.azure_communication_ui_calling_view_share_diagnostics,
+                R.drawable.azure_communication_ui_calling_ic_fluent_share_android_24_regular),
+            REPORT_ISSUE(
+                R.string.azure_communication_ui_calling_report_issue_title,
+                R.drawable.azure_communication_ui_calling_ic_fluent_person_feedback_24_regular),
+        }
+    }
```

Comment:
Suggestion: I think this should be moved to view as it required reference to view or activity.

Reply:
These imports are actually unused. I'll delete.

The R.id's are fed up via the view model, but the actual context/getString is in the view.